pages/1.7/release-notes/1.7.2/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.7.2
+title: Release Notes for 1.7.2
+menuWeight: 2
+excerpt:
+
+enterprise: false
+---
+
+The release notes provide a list of useful topics and links for DC/OS.
+
+# <a name="known-issues"></a>Known Issues and Limitations
```

Comment:
Is there a reason we carried over the Known Issues but not the features from 1.7? Either we should list all the features of the major release (including known issues) or we should cross-check the known issues with the issues fixed in 1.7.2 and note that these are known issues with 1.7.2 specifically.